Installer/config/FanartHandler.Holidays.xml

```diff
@@ -435,9 +512,11 @@
 	<Holiday name="Daylight Savings Time Begins" shortname="DaylightSavings|DaylightSavingsBegin">
 		<DSTStart />
 		<NameDE>Beginn der Sommerzeit</NameDE>
+		<NotCelebratedIn>AU</NotCelebratedIn>

```

Comment:
Once you make changes to transfer to Custom, it is better to remove this.
And from Daylight Savings Time Ends to ...